libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -643,7 +643,10 @@ void AC_PosControl::update_xy_controller()
     // Position Controller
 
     const Vector3f &curr_pos = _inav.get_position_neu_cm();
-    Vector2f vel_target = _p_pos_xy.update_all(_pos_target.x, _pos_target.y, curr_pos);
+    Vector3f comb_pos = curr_pos;
+    comb_pos.x += _disturb_pos.x;
+    comb_pos.y += _disturb_pos.y;
```

Comment:
I think `comb_pos.xy() += _disturb_pos;` will work.

Reply:
I only pass in an xy disturbance, not xyz.  So I have to breakout the assignment to the 3 axis vector.

Reply:
Done. Thanks